### PR TITLE
Committing files

### PR DESCRIPTION
**Pull request by vtaneka**
For the implementation of the minimum attribute task launch

### CODE REVIEW

File: server/libs/skills/ac/createcrosstabquery/createCrossTabQuery.js [hunk 1]
```diff
@@ -18,8 +18,10 @@ class createcrosstabquery extends AccessBaseSkill {
                 this.finalQueryName = defaultQueryName;
          }
          return Promise.resolve(true);
-         }catch(error){
-            return Promise.reject(true);
+        }catch(error){
+            console.log("Error Occured in the init function: Still resolving");s
```
Comment (dhirendra-singh81) [inline server/libs/skills/ac/createcrosstabquery/createCrossTabQuery.js]: remove s

File: server/libs/skills/pp/common/ppSkill.js [hunk 2]
```diff
@@ -3,6 +3,14 @@ const BaseSkill = require("../../common/baseSkill"),
     DOMParse = require("xmldom").DOMParser;
 
 module.exports = class PPTBaseSkill extends BaseSkill {
+
+    constructor() {
+        super();
+        /* Creating a list of he mandatory initial attributes which are application specific, these are mandatorily needed 
+        to succeed in the AttrValueMap generation */
+        this.mandatoryAttributeList = ["XML_PATH"];
```
Comment (dhirendra-singh81) [inline server/libs/skills/pp/common/ppSkill.js]: attribute for adding slide images should also be mandatory. Check with @punit01

File: server/libs/skills/wd/common/wdSkill.js [hunk 3]
```diff
@@ -6,6 +6,14 @@ const xmlUtil = require("../../../../utils/xmlUtil");
 const dbfileStoreManager = require('../../../../modules/skill/dbFilestoreMgr');
 
 module.exports = class WordSkill extends BaseSkill {
+    
+    constructor() {
+       super();
+       /* Creating a list of he mandatory initial attributes which are application specific, these are mandatorily needed 
+        to succeed in the AttrValueMap generation */
+       this.mandatoryAttributeList = ["DOCUMENT_DATA"];
```
Comment (dhirendra-singh81) [inline server/libs/skills/wd/common/wdSkill.js]: task xml attribute to be provided here, @aditi-kapur to check

Comment (aditi-kapur) [reply server/libs/skills/wd/common/wdSkill.js]: HTML_PATH

**Review by dhirendra-singh81 — APPROVED** [review]

**Pull request merged**